lib/services/game_library_S/game_service.dart

```diff
@@ -3,18 +3,69 @@ import 'dart:io';
 import '../../model/game_library_M/game_details_model.dart';
 import 'package:http/http.dart' as http;
 
-class GameService{
-Future<GameDetails> fetchGameDetails( gameID) async {
-  try {
-    final response = await http.get(Uri.parse(
-        'https://api.rawg.io/api/games/$gameID?key=b8d81a8e79074f1eb5c9961a9ffacee6'));
-    if (response.statusCode == 200) {
-      return GameDetails.fromJson(jsonDecode(response.body));
-    } else {
-      throw Exception('Failed to load game details');
+class GameService {
+  final String apiKey = 'b8d81a8e79074f1eb5c9961a9ffacee6';
```

Comment:
just a suggestion, because I saw gitguardian found this key... maybe move the api key to the .env file and use it that way. When you do this, please also let us know on the group, as we will have to update the .env file github workflow uses.

Reply:
I put the api key in the .env file.